config/routes.rb

```diff
@@ -8,9 +8,18 @@
   # Example of regular route:
   #   get 'products/:id' => 'catalog#view'
 
+  resources :donors
+  resources :donations do
```

Comment:
The only routes that should be defined as part of this feature are:

- `donations#create`
- `donor#show`